internal/backends/winit/build.rs

```diff
@@ -11,4 +11,10 @@ fn main() {
        skia_backend_metal: { all(target_os = "macos", not(feature = "renderer-winit-skia-opengl")) },
        skia_backend_d3d: { all(target_family = "windows", not(feature = "renderer-winit-skia-opengl")) },
     }
+
+    println!("cargo:rerun-if-env-changed=RUST_FONTCONFIG_DLOPEN");
+    let dlopen = std::env::var("RUST_FONTCONFIG_DLOPEN").is_ok();
+    if dlopen {
+        println!("cargo:rustc-cfg=feature=\"fontconfig-dlopen\"");
```

Comment:
This should be a feature, just a plain cfg.

Also, could we have the default reversed? Default to dlopen and have a env variable to disable it?

Reply:
> This should be a feature, just a plain cfg.

I'm not sure what you mean. Listing it as a feature in Cargo.toml? I could do that, but it requires doing that in every crate that uses fontconfig for building the workspace with `--all-features` to work.

> Also, could we have the default reversed? Default to dlopen and have a env variable to disable it?

Technically possible, but that would require changing how the build scripts handle the environment variable all up and down the stack (yeslogic-fontconfig-sys, fontconfig, fontkit, plotters) which I'd rather not deal with.

Reply:
I mean `println!("cargo:rustc-cfg=fontconfig-dlopen");`.  Because i believe the `feature=`  is to be used with cargo features. 
Unless i'm missing something. What other crate are using that.


> Technically possible, but that would require changing how the build scripts handle the environment variable all up and down the stack (yeslogic-fontconfig-sys, fontconfig, fontkit, plotters, Slint crates) which I'd rather not deal with.

I see. This is unfortunate.

Reply:
`feature=` is required for [dlib::ffi_dispatch!](https://docs.rs/dlib/0.5.0/dlib/macro.ffi_dispatch.html).